app/models/item.rb

```diff
@@ -11,7 +11,7 @@ class Item < ActiveRecord::Base
     self.name = (sign.is_a?(Sign) ? sign.gloss_main : nil) if name.nil?
     self.maori_name = (sign.is_a?(Sign) ? sign.gloss_maori : nil) if maori_name.nil?
     self.sign_id = sign.id if sign_id.nil?
-    self.drawing = sign.drawing
+    self.drawing = sign.drawing if drawing.nil?
```

Comment:
is this still needed?

Reply:
I liked the consistency, but I can revert it